independent-publisher-connectors/Placedog/apiDefinition.swagger.json

```diff
@@ -0,0 +1,187 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Placedog",
+    "description": "Request dogs from placedog.net. Hand-picked, stunning photographs of our favorite pets: dogs! Free to use with your websites/projects! Featured in the #ThePawryParty Chrome Extension!",
+    "version": "1.0",
+     "contact": {
+      "name": "Troy Taylor",
+      "url": "https://www.hitachisolutions.com",
+      "email": "ttaylor@hitachisolutions.com"
+     }
+  },
+  "host": "placedog.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/{width}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}

```

Comment:
@troystaylor Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
@troystaylor Applies the same comment for empty schemas in the swagger.